### PR TITLE
Mobile first Portfolio set  

### PR DESCRIPTION
**Pull request by lengushuru**
## description 

updated index.html and styles.css files as per the reviewer's feedback.
the main tag was replaced with a section tag, social media icons were made to list items. only the nav-bar to be enclosed by header tag.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            <div class="page--logo">
+                <a class="page--link" href="#">My Logo</a>
+            </div>
+            <div class="nav--menu">
+                    <div></div>
+                    <div></div>
+                    <div></div>
+            </div>
+        </nav>
+        <main class="main--content">
```
Comment (nedjwab) [inline index.html:25]: [ ] kindly use a `section` tag for the headline.It represents a generic standalone section of the document, so use it. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section) for more info.🤓

File: index.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+             <div class="about--container">
```
Comment (nedjwab) [inline index.html:26]: [ ] kindly move this section outside the `header` tag. the header tag should be used just for the Toolbar.🤓

File: index.html [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                <h1>I’m Charles<br>Glad to see<br> you!</h1> 
+             </div> 
+             <div class="description--container">
+                <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+             </div>
+             <p class ="supporting--text">LET’S CONNECT</p>
+             <div class="social--icons">
```
Comment (nedjwab) [inline index.html:33]: [ ] kindly use a `ul` tag to group the icons list, and `li` per icon.🤓

**Review by nedjwab — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @lengushuru 

Nice job, you are almost there.



## (Highlights) Good Points: :
- Good implementation of some of the project requirements  ✔
- GitHub flow was correctly used ✔.
- Nice design 🔥🔥🔥🔥

## (Changes Required) Aspects to improve: ♻️
You did great implementing some of the  project requirements, but you still have some issues: 

` [ ] Pull Request should have a short summary describing introduced changes in more detail. kindly ffix that 🤓`
check the comments below 👇👇
 
## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html (hunk 1 above)
Comment (lengushuru) [reply index.html:25]: thank you @nedjwab but I don't quite understand this point.  should I replace <main> tag with <section> tag?

File: index.html (hunk 3 above)
Comment (lengushuru) [reply index.html:33]: @nedjwab well understood I will fix this ASAP.

File: index.html (hunk 2 above)
Comment (lengushuru) [reply index.html:26]: @nedjwab well understood I will fix this ASAP.

File: index.html (hunk 1 above)
Comment (nedjwab) [reply index.html:25]: > thank you @nedjwab but I don't quite understand this point. should I replace the main tag with the section tag?

kindly replace the` main ` tag with a ` section` tag 🤓

Comment (lengushuru) [reply index.html:25]: @nedjwab thank you for your time I will make the required changes.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+        </div>
+        <p class ="supporting--text">LET’S CONNECT</p>
+        <div class="social--icons">
+            <ul>
+                <li><a href="#"><img src="images/tweeter-icon.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Linkedin-icon.png" alt=""></a></li>
+                <li><a href="#"><img src="images/Vector(2).png" alt=""></a></li>
+                <li><a href="#"><img src="images/github.png" alt=""></a></li>
+                <li><a href="#"><img src="images/peace-icon.png" alt=""></a></li>
+            </ul>
```
Comment (anagudelogu) [inline index.html:41]: - [ ] **[OPTIONAL]:** Great job implementing the suggested changes by the previous reviewer! 💯 Please consider adding alternative text for each one of these icons, so the assistive technologies such as screen readers can have a better time browsing your site! Also, kudos ⭐  if you include your social media links so people who see your portfolio can connect with you by just clicking them! 🤓

**Review by anagudelogu — APPROVED** [review]
Hi @lengushuru, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
✅ Great design! 💯 
✅ Linters are passing! 👍🏼 
✅ Descriptive commits. 🥇 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**